packages/core/content-type-builder/admin/src/components/AttributeOptions/AttributeOption/index.js

```diff
@@ -29,15 +39,27 @@ const AttributeOption = ({ type }) => {
     });
   };
 
+  // TODO: Remove after the RTE Blocks Alpha release
+  const alphaTagStylesWidth = type === 'blocks' && { width: '100%' };
+  const alphaTagStylesJustifyContent = type === 'blocks' && { justifyContent: 'space-between' };
+
```

Comment:
ℹ️  This is probably overkill but I just thought I would try to avoid applying useless styles to other elements.

Reply:
I get what you mean but I would rather keep it the code simple when possible. I don't think a couple of non essential styles are problematic

Reply:
works for me, I'll remove it.